stable/elasticsearch-exporter/templates/deployment.yaml

```diff
@@ -48,13 +48,22 @@ spec:
       {{- end }}
       containers:
         - name: {{ .Chart.Name }}
-          {{- if .Values.env }}
```

Comment:
If .Values.env is not defined and .Values.envFromSecrets is not defined, `env` will be empty and will not respect kubernetes schemas.

I suggest putting a `or` condition there.

Reply:
That should not affect release even with empty `env` parameter, but for now, it does not matter due to the changing approach of setting env from a secret